app/views/PartTimeHoursView.scala.html

```diff
@@ -0,0 +1,52 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import pages.PartTimeHoursPage
+
+@this(
+        govukLayout: templates.GovukLayoutWrapper,
+        formHelper: FormWithCSRF,
+        errorSummary: components.errorSummary,
+        button: components.button,
+        h1: components.h1,
+        p: components.p,
+        inputNumber: components.inputNumber
+)
+
+@(form: Form[_], partTimePeriod: Period, idx: Int)(implicit request: Request[_], messages: Messages)
```

Comment:
as above, i think this wants to be of type Periods, and then we need conditional content. Maybe the prototype hasn't been updated yet but I thought we agreed to show how many hours have they worked between $start and $end